python/lsst/ctrl/bps/wms/panda/edgenode/sw_runner

```diff
@@ -4,6 +4,10 @@
 cd /tmp;
 export HOME=/tmp;
 export S3_ENDPOINT_URL=https://storage.googleapis.com
+export AWS_ACCESS_KEY_ID=$(</cloudcreds/SINGULARITYENV_AWS_ACCESS_KEY_ID)
+export AWS_SECRET_ACCESS_KEY=$(</cloudcreds/SINGULARITYENV_AWS_SECRET_ACCESS_KEY)
+export PGPASSWORD=$(</cloudcreds/SINGULARITYENV_PGPASSWORD)
```

Comment:
Fine to merge as is as other keys are also being saved in environment variables.  I was just wondering if there was a reason to use PGPASSWORD instead of PGPASSFILE (which is more secure and what PostgreSQL documentation recommends instead of PGPASSWORD)?

Reply:
It was just one way it works. If we use PGPASSFILE instead, something has to take care of the password file then.